app/sdk/cards/cardSetFactory.coffee

```diff
@@ -1,9 +1,10 @@
-
 CardSet = require './cardSetLookup'
 i18next = require 'i18next'
 
-class CardSetFactory
+defaultOrbGoldCost = 50
+defaultFullSetSpiritCost = 7500
```

Comment:
This may have been based on the 300 Gold Orb price, so maybe this should actually be 2500. Need to count the actual Spirit value of crafting one at a time to verify.